Presentation/Presentation/Sources/WhiteboardList/View/WhiteboardListViewController.swift

```diff
@@ -94,16 +102,25 @@ public final class WhiteboardListViewController: UIViewController {
 
         let createWhiteboardAction = UIAction { [weak self] _ in
             self?.viewModel.action(input: .createWhiteboard)
+
+            guard
+                let whiteboardViewModel = self?.whiteboardViewModel,
+                let whiteboardObjectViewFactory = self?.whiteboardObjectViewFactory
+            else { return }
+
+            let whiteboardViewController = WhiteboardViewController(viewModel: whiteboardViewModel,
+                                                                    objectViewFactory: whiteboardObjectViewFactory)
```

Comment:
단순 스타일입니다만, 아래 컨벤션이 저희가 사용하는 컨벤션입니다!

```suggestion
            let whiteboardViewController = WhiteboardViewController(
                viewModel: whiteboardViewModel,
                objectViewFactory: whiteboardObjectViewFactory)
```

Reply:
수정했습니다 감사합니다!